src/main/java/org/jenkinsci/test/acceptance/plugins/credentials/StringCredentials.java

```diff
@@ -9,7 +9,7 @@
 @Describable("Secret text")
 public class StringCredentials extends BaseStandardCredentials {
 
-    public Control secret = control("secret");
+    public Control secret = control(by.name("_.secret"));
```

Comment:
What changes from 1.x to 2.x is the `path` attribute. `control` by default uses the `path` attribute.

What has NOT changed from 1.x to 2.x is the `name` attribute. So, the simplest way to make this work with both 1.x and 2.x lines is to look by `name` instead of by `path`.

Answering your question, yes, works for both.